staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/validator_test.go

```diff
@@ -819,11 +832,46 @@ func TestValidate(t *testing.T) {
 			},
 			costBudget: 1, // shared between expression and messageExpression, needs 1 + 1 = 2 in total
 		},
+		{
+			name:    "no match surpresses failure",
+			matcher: &fakeCELMatcher{matches: false},
+			evaluations: []cel.EvaluationResult{
+				{
+					Error:              errors.New(""),
```

Comment:
nit: put some error text in, maybe `"expected"`, in case it gets logged somewhere.

Reply:
done